docs/api/read/programme.md

```diff
@@ -27,134 +75,182 @@ A programme is composed by a set of _offers_.
 ## Link Relations
    * A programme representation
       - must include a link to its context, using the self link relation.
-      - may include a link to the collection it belongs, using the collection link relation.
-      - must include a link to its _offers_.
-
-## Actions
-    * edit: edit's a programme
-    * add Offer: add's a curricular unit offer to the programme
+      - must include a link to the collection it belongs, using the `collection` relation.
+      - must include a link to the offers collection, using the `/rel/offers` relation.
 
-## Example representation
+## Example Representation
 ```json
 {
-    "class": [ "programme" ],
+    "class": ["programme"],
     "properties": {
         "id": 1,
-        "name": "Licenciatura em Engenharia Informática e de Computadores", 
+        "name": "licenciatura eng. inf.",
         "acronym": "LEIC",
         "termSize": 6
     },
     "entities": [
         {
-            "class": [ "offer" ],
-            "title": "LS Offer",
-            "rel": [ "/rel/programmeOffer" ],
+            "class": ["offer"],
+            "rel": ["/rel/programmeOffer"],
+            "title": "Software Laboratory",
             "properties": {
                 "id": 1,
+                "acronym": "SL",
                 "courseId": 2,
-                "termNumber": 3
+                "termNumber": [3]
```

Comment:
This is a breaking change. We probably should have a document with those.

Reply:
The API already provided the array with the term numbers, therefore this was only a documentation mistake and not a breaking change.

The client is handling this array as shown in [this example](https://github.com/i-on-project/android/blob/master/Project/app/src/main/java/org/ionproject/android/programmes/ProgrammeMapper.kt#L47).